akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowIteratorSpec.scala

```diff
@@ -39,9 +39,7 @@ class FlowIterableSpec extends AbstractFlowIteratorSpec {
     sub.request(1)
     c.expectNext(1)
     c.expectNoMsg(100.millis)
-    EventFilter[IllegalStateException](message = "not two", occurrences = 1).intercept {
```

Comment:
Just removing as it is tested in the expectError?

Reply:
it's not logged any more.

Logging is done if you throw and exception from a GraphStage, but not if you `failStage`.